src/actions/index.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import { LINKS, FETCH_PROFILE } from "../constants/actionTypes";
+
+type Link = {
+  url: string,
+  shortLink: string,
+  passage: number,
+  tags: Array<string>,
+  description: string
+};
+type LoadMy = { type: typeof LINKS.LOAD_MY };
+type LoadAll = { type: typeof LINKS.LOAD_ALL };
+type Request = { type: typeof LINKS.REQUESTED };
+type RequestSuccess = {
+  type: typeof LINKS.REQUESTED_SUCCEEDED,
+  data: Array<Link>
+};
+type RequestError = { type: typeof LINKS.REQUESTED_FAILED, error: string };
+
+export const links = {
```

Comment:
Please, export action creators by 1 method, in such case you'll avoid importing unnecessary creators where it shouldn't be imported